src/tests/mutation_log_learn.cpp

```diff
@@ -138,7 +138,7 @@ TEST(replication, log_learn)
             ).end_tracking();
         auto time_toc = clock.now();
         
-        std::cout << "TEST replay time(us): " << std::chrono::duration_cast<std::chrono::microseconds>(time_toc - time_tic).count() << std::endl;
+        std::cout << "TEST replay time(ms): " << std::chrono::duration_cast<std::chrono::microseconds>(time_toc - time_tic).count() << std::endl;
```

Comment:
sure, because I run log_learn test and it took a long time (about 30 seconds), so I thought maybe the timer is in ms. My fault, I should have look into it more carefully.
Now I find it's the writing step which takes so long a time, and I have improved it to less than 1 second.
btw, in mutation_log::flush(), the dsn_task_tracker_wait_all(tracker()) takes about 25ms to wait, is it reasonable? maybe we need test its performance and find out something wrong...